pkg/descriptions.json

```diff
@@ -1,2074 +1,2081 @@
-      "System.Runtime.Serialization.DataContractResolver",
-      "System.Xml.IXmlDictionary"
-    ]
-  },
-  {
-    "Name": "System.Runtime.WindowsRuntime",
-    "Description": "Provides extensions methods that improve interoperation between managed code and the Windows Runtime.",
-    "CommonTypes": [
-      "System.WindowsRuntimeSystemExtensions",
-      "System.IO.WindowsRuntimeStorageExtensions",
-      "System.IO.WindowsRuntimeStreamExtensions",
-      "System.Runtime.InteropServices.WindowsRuntime.AsyncInfo",
-      "System.Runtime.InteropServices.WindowsRuntime.WindowsRuntimeBuffer"
-    ]
-  },
-  {
-    "Name": "System.Runtime.WindowsRuntime.UI.Xaml",
-    "Description": "Provides managed equivalents of Extensible Application Markup Language (XAML) UI types from Windows Runtime.",
-    "CommonTypes": [
-      "Windows.UI.Xaml.CornerRadius",
-      "Windows.UI.Xaml.Duration",
-      "Windows.UI.Xaml.DurationType",
-      "Windows.UI.Xaml.GridLength",
-      "Windows.UI.Xaml.GridUnitType",
-      "Windows.UI.Xaml.Thickness",
-      "Windows.UI.Xaml.Media.Matrix",
-      "Windows.UI.Xaml.Media.Animation.KeyTime",
-      "Windows.UI.Xaml.Media.Animation.RepeatBehavior",
-      "Windows.UI.Xaml.Media.Animation.RepeatBehaviorType",
-      "Windows.UI.Xaml.Media.Media3D.Matrix3D"
-    ]
-  },
-  {
-    "Name": "System.Security.AccessControl",
-    "Description": "Provides base classes that enable managing access and audit control lists on securable objects.",
-    "CommonTypes": [
-      "System.Security.AccessControl.AccessRule",
-      "System.Security.AccessControl.AuditRule",
-      "System.Security.AccessControl.ObjectAccessRule",
-      "System.Security.AccessControl.ObjectAuditRule",
-      "System.Security.AccessControl.ObjectSecurity"
-    ]
-  },
-  {
-    "Name": "System.Security.Claims",
-    "Description": "Provides classes that implement claims-based identity in the .NET Framework, including classes that represent claims, claims-based identities, and claims-based principals.",
-    "CommonTypes": [
-      "System.Security.Principal.GenericIdentity",
-      "System.Security.Claims.Claim",
-      "System.Security.Claims.ClaimsIdentity",
-      "System.Security.Claims.ClaimsPrincipal",
-      "System.Security.Principal.GenericPrincipal"
-    ]
-  },
-  {
-    "Name": "System.Security.Cryptography.Algorithms",
-    "Description": "Provides base types for cryptographic algorithms, including hashing, encryption, and signing operations.",
-    "CommonTypes": [
-      "System.Security.Cryptography.Aes",
-      "System.Security.Cryptography.RSA",
-      "System.Security.Cryptography.RSAParameters",
-      "System.Security.Cryptography.HMACSHA1",
-      "System.Security.Cryptography.SHA256",
-      "System.Security.Cryptography.SHA1",
-      "System.Security.Cryptography.SHA512",
-      "System.Security.Cryptography.SHA384",
-      "System.Security.Cryptography.HMACSHA256",
-      "System.Security.Cryptogr
```

Comment:
Did you convert this file to having tabs? Or did you just resize all the tabs? In general we try to match the formatting of the file as opposed to resetting the entire thing. As it stands it is hard to see what you actually changed in this diff.